website/docs/r/spring_cloud_service.html.markdown

```diff
@@ -137,6 +153,12 @@ The following attributes are exported:
 
 * `id` - The ID of the Spring Cloud Service.
 
+---
+
+The `network` block exports the following:
+
+* `load_balancer_ip` - The Load Balancer IP for Azure Spring Cloud Service Runtime.
```

Comment:
what does this mean in practice?

Reply:
'app_endpoint_ip' - The IP address of customer's Spring Boot microservices Applications which deployed in the Azure Spring Cloud Service Instance.

How do you think

Reply:
after discussed with bowen5, I have deleted this field.